src/PublicApiAnalyzers/Core/Analyzers/PublicApiAnalyzerResources.resx

```diff
@@ -208,12 +208,6 @@
   <data name="ImplicitSetAccessor" xml:space="preserve">
     <value>implicit set-accessor for '{0}'</value>
   </data>
-  <data name="PublicApiFilesMissingTitle" xml:space="preserve">
-    <value>One or both of the public API files are missing</value>
-  </data>
-  <data name="PublicApiFilesMissingMessage" xml:space="preserve">
-    <value>The solution must contain two files with the type "AdditionalFiles": PublicAPI.Unshipped.txt and PublicAPI.Shipped.txt. At least one of these files is missing or has the wrong type.</value>
```

Comment:
Hmm I'd thought I had added a diagnostic for this case.

Reply:
Ah, I see there are duplicate resources and the new ones without an 's' are being used: https://github.com/dotnet/roslyn-analyzers/blob/34396c7d144cc3eea6ce5db952fa564edfd45f98/src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer_Diagnostics.cs#L175-L193